### PR TITLE
yuzu: Add version 0-661

### PR DESCRIPTION
**Pull request by NoraTheGamer**

### CODE REVIEW

**Comment by Calinou** [comment]
Out of curiosity, why is the package called `yuzu-mainline` instead of `yuzu`?

**Comment by NoraTheGamer** [comment]
Yuzu used to have seperate branches like canary and nightly but were since discontinued, there is also the patreon branch. Feel free to rename to just yuzu

**Comment by Calinou** [comment]
I agree the package should renamed to `yuzu` – I prefer the mainline package to have the shortest possible name. Feel free to do it in this PR :slightly_smiling_face:

**Comment by NoraTheGamer** [comment]
There you go

File: bucket/yuzu.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://yuzu-emu.org/",
+    "version": "0-661",
```
Comment (Calinou) [inline bucket/yuzu.json:3]:
```suggestion
    "homepage": "https://yuzu-emu.org/",
    "description": "Experimental emulator for the Nintendo Switch",
    "version": "0-661",
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**